packages/core/src/browser/dialogs.ts

```diff
@@ -23,6 +23,8 @@ import { FrontendApplicationContribution } from './frontend-application';
 @injectable()
 export class DialogProps {
     readonly title: string;
+    maxWidth?: number;
```

Comment:
I think we should document this prop to be more specific such as:

```ts
/**
* Determines the maximum width of the dialog in pixels.
*/
maxWidth?: number;
```

Reply:
yes, also what happens if there are not provided